CodeStarterCamp_Week2/main.swift

```diff
@@ -8,5 +8,31 @@
 
 import Foundation
 
-print("Hello, World!")
+func createRandomNumber() -> Int {
+        return Int.random(in: 1..<46)
+}
+var winnerNumber: Set<Int> = Set<Int>()
+func createWinnerNumbers() -> Set<Int> {
+    repeat {
+        winnerNumber.insert(createRandomNumber())
+    } while winnerNumber.count < 6
+    
+    return winnerNumber
+}
+createWinnerNumbers()
 
+let myLottoNumbers: [Int] = [7, 12, 15, 22, 43, 44]
+func createLottoNumberIndex(indexNum: Int) -> Int {
+    for indexNumber in 0...indexNum{
```

Comment:
```suggestion
    for indexNumber in 0...indexNum {
```
for문 내 컨벤션을 알려드립니다..!!